res/qml/main.qml

```diff
@@ -274,6 +274,7 @@ ApplicationWindow {
       onClicked: {
         vis.forceActiveFocus()
         instantiate(algorithmSelectBox.currentText)
+        vis.focusOnCenterOfMass();
```

Comment:
I would remove this "fix". It adds the annoyance of resetting the camera to center even if the user has already figured out some view they like more (e.g., offset a bit to the left to move it out from under the sidebar).